Blackbaud.CustomFx.ConstituentPhotoAnalysis.Catalog/ConstituentPhotoAnalysis.Package.xml

```diff
@@ -0,0 +1,30 @@
+<PackageSpec
```

Comment:
Do we want to use the package spec to make changes to the out-of-box Constituent Page rather than making an all new one?

Reply:
I didn't include the Constituent Page spec in the package for that reason. I also put a warning on the readme about loading the page.